docs/backend/configuration.md

```diff
@@ -40,3 +41,43 @@ object and viewed / modified via `/internal/config` (see
 [Accessing the admin panel](./admin.md)).
 
 ![Screenshot of site configuration admin interface](https://user-images.githubusercontent.com/47985/73627238-6276d500-467e-11ea-8724-afb703f056bc.png)
+
+## Vault
+
+The vault Ruby gem allows us to interact with
+[HashiCorp's Vault](https://www.vaultproject.io/docs/what-is-vault). In a
+nutshell, Vault is a tool for securely storing and accessing secrets. The way we
+have it configured, it is completely optional for running a Forem. To access it
+we use the wrapper `AppSecrets`.
+
+```ruby
+class AppSecrets
+  def self.[](key)
+    result = Vault.kv(namespace).read(key)&.data&.fetch(:value) if ENV["VAULT_TOKEN"].present?
+    result ||= ApplicationConfig[key]
+
+    result
+  rescue Vault::VaultError
+    ApplicationConfig[key]
+  end
+
+  def self.[]=(key, value)
+    Vault.kv(namespace).write(key, value: value)
+  end
+
+  def self.namespace
+    ENV["VAULT_SECRET_NAMESPACE"]
+  end
+  private_class_method :namespace
+end
+```
+
+As you can see in the code, we attempt to access a secret from Vault if it is
```

Comment:
```suggestion
We attempt to access a secret from Vault if it is
```

Might not be evident to someone who is more of a beginner, and they might read "as you can see" and wonder why they don't see. If you wanted to keep it in, I'd recommend pointing to the line where this is happening to make it more clear. But, I think it's also fine to just ✂️ it, too.